examples/models/txtl_model_bioscrape.xml

```diff
@@ -12,6 +12,8 @@
 <parameter name="DummyVar_PositiveHillPropensity_n_0" value="1.0" />
 <parameter name="ktl" value="10.0" />
 <parameter name="delta" value="0.1" />
+<parameter name="_I0" value="nan" />
+<parameter name="_T_I0" value="nan" />
```

Comment:
Just wanted to note that I see underscores here. Might be related to a different issue.

Reply:
just fixed this